### PR TITLE
fix random color #111

### LINKED ISSUE #111: We got a random color again.
We got a random color again.
rnd

_Originally posted by @dzianisshupenka in https://github.com/juggler39/ais-chess/pull/106#issuecomment-709838377_

### PR DESCRIPTION
**Pull request by dzianisshupenka**
Now it works, please check it on your side.
closes #111

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -171,16 +171,16 @@ socketIO.on('connection', (socket) => {
               OpenGame.find({ isOpen: true }, (err, allOpenGames) => {
                 socketIO.sockets.emit('newGameInfo', allOpenGames);
               });
+            }).then(() => {
+              OpenGame.find({ _id: player2info.gameId }, (err, game) => {
+                socket.join(player2info.gameId);
+                console.log(game);
```
Comment (TapeGhad) [inline backend/app.js]: Yes, looks good, but console,log is not needed anymore

**Review by TapeGhad — CHANGES_REQUESTED** [review]
Remove console log

File: backend/app.js (hunk 1 above)
Comment (juggler39) [reply backend/app.js]: I agree, remove it.

**Review by juggler39 — CHANGES_REQUESTED** [review]

File: backend/app.js (hunk 1 above)
Comment (dzianisshupenka) [reply backend/app.js]: sorry)

**Review by juggler39 — APPROVED** [review]
Looks good now.

**Pull request merged**